Fix automation action selector popup on small devices [MAILPOET-5184]

NeosinneR opened the pull request:
## Description

Add a `min-width` for automation action selector popup, otherwise it will be too narrow due to `width: min-content` property. 

## Code review notes

_N/A_

## QA notes

QA can be skipped. 

## Linked PRs

_N/A_

## Linked tickets

[MAILPOET-5184]

## After-merge notes

_N/A_

## Tasks

- [x] 🚫 I followed [best practices](https://codex.wordpress.org/I18n_for_WordPress_Developers) for translations
- [x] 🚫 I added sufficient test coverage
- [x] 🚫 I embraced TypeScript by either creating new files in TypeScript or converting existing JavaScript files when making changes


[MAILPOET-5184]: https://mailpoet.atlassian.net/browse/MAILPOET-5184?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

NeosinneR committed:
Fix automation action selector popup on small devices

[MAILPOET-5184]

costasovo reviewed (commented):
@NeosinneR  I tried it locally. It is way better. 
But on iPhone Max sized and smaller displays the popup doesn't fit the screen and it causes strange infinite horizontal scrolling. Please see the screenshot.

One way we could prevent it is setting `position={'middle center bottom'}` to the inserter popup component. There is [wordpress/viewport](https://github.com/WordPress/gutenberg/tree/0b94a19ca3307f98fc7124573e8be51cbe32cfd6/packages/viewport) package that allows to detect screen size in react. So with that we could center the popup only on small screens.

![Screenshot 2024-07-24 at 13 46 01](https://github.com/user-attachments/assets/ef83241d-8f0f-4348-acc9-57703cf3e3da)

NeosinneR committed:
Show automation action inserter popup in the bottom center position

This fixes position especially on mobile phones, but also on desktops < ~1050px

[MAILPOET-5184]

NeosinneR committed:
Replace deprecated prop

[MAILPOET-5184]

NeosinneR commented:
@costasovo great suggestion. It turned out that the position is not the best even on a 1050px wide desktop, so I just centered it everywhere, not just on mobiles.

FYI, `position` is [deprecated](https://developer.wordpress.org/block-editor/reference-guides/components/popover/#position-yaxis-xaxis-optionalcorner), so I used `placement` instead.

<img width="1079" alt="Screenshot 2024-07-24 at 21 07 40" src="https://github.com/user-attachments/assets/47656d23-07d1-414e-939f-d8026a72ddb9">

costasovo reviewed (approved):
Nicely done! Thank you!
